cli/azd/CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # Release History
 
-## 1.11.0-beta.1 (Unreleased)
+## 1.11.0 (2024-11-13)
 
 ### Features Added
 
-### Breaking Changes
+- [[4527]](https://github.com/Azure/azure-dev/pull/4527) Add command `azd add`.
```

Comment:
- Callout as alpha in the description?
- Can we also add it to the features page marking as Alpha?

Reply:
features page you mean> https://github.com/Azure/azure-dev/pull/4546/files#diff-f101ce6ad2b55d90194098ab21ff35e156e9db2a3378c016650f4a98ee4d34baR22 ?